### ISSUE TITLE
use wrong place for mathjax.config.js

### ISSUE THREAD

**Issue by fanhongtao** [gollum/gollum#1602]
Beacuse  #1572 , I run Gollum by [RUNNING FROM SOURCE](https://github.com/gollum/gollum#running-from-source).

* When I write a page in the root directory, Gollum uses `mathjax.config.js` in the root.
* When I write a page in the a sub-directory, Gollum tries to use `mathjax.config.js` in that sub-directory. 

If I run Gollum 5.0.1, Gollum always uses `mathjax.config.js` in the root.

After read the git logs, I think the problem maybe introduced [Use page_route() for custom.css, custom.js, and MathJax ](https://github.com/gollum/gollum/commit/6e6d9d8c1fbeca072c2b7a14208d808757bd5b44#diff-ae57601ec9f43453506032bb4295c44d)

And I fix the problem on my computer by change `layout.mustache` from 

```html
<script type="text/javascript" src="{{mathjax_config}}"></script>
```

to 

```html
<script type="text/javascript" src="/{{mathjax_config}}"></script>
```

Don't know if it's appropriate.

 @dometto

**Comment by heavywatal** [comment]
Thank you for the report. You are right. I have submitted a quick fix for this.

**Comment by dometto** [comment]
Fixed by @heavywatal in #1604. Will do a gollum bugfix release soon

**Comment by dometto** [comment]
Actually, fixing this required another change: https://github.com/gollum/gollum/pull/1605

**Comment by dometto** [comment]
Released in 5.1
